src/Components/Container/FormContainer.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import Button from '../Form/Button';
+import DatePicker from '../Form/DateOfBirth';
+import Error from '../ErrorPrompt/Error';
+import InputText from '../Form/InputText';
+import Label from '../Form/Label';
+import PassStrengthBar from '../Form/PassStrenghtBar';
+import RadioButton from '../Form/RadioButton';
+import SelectOption from '../Form/SelectOption';
+import './Form-container.scss';
+
+const FormContainer = () => {
+  // state for containing data
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [dateOfBirth, setDateOfBirth] = useState('');
+  const [gender, setGender] = useState('');
+  const [education, setEducation] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  // change handler for states
+  const changeHandlerName = (e) => {
+    setName(e.target.value);
```

Comment:
Use forms on submit an event to get values instead of using state for each form element